Magento_Theme/templates/header/minicart.phtml

```diff
@@ -0,0 +1,69 @@
+<?php
+/** @var $block \Magento\Checkout\Block\Cart\Sidebar */
+?>
+
+<div class="header__minicart minicart-wrapper" data-block="minicart">
+    <a class="minicart-trigger button action showcart minicart-trigger"
+       href="<?= /* @escapeNotVerified */ $block->getShoppingCartUrl() ?>"
+       data-bind="scope: 'minicart_content'"
```

Comment:
I'd add an `aria-label` here but text should be: "Shopping cart link, You have <counter> in shopping cart". I don't know it is easy to do to bind this and love it with knockout, if it is, please add, if it's too much work for now, please add it to the: [#133](https://github.com/SnowdogApps/magento2-alpaca-components/issues/133)